RELEASE.md

```diff
@@ -25,135 +35,59 @@ gh auth login
 ? How would you like to authenticate GitHub CLI? Login with a web browser
 ```
 
-Releasing
----------
-
-1.  Set up some environment variables for use later.
-
-    ```
-    # This is the version being released.
-    VERSION=1.0.0
-
-    # This is the branch from which $VERSION will be released.
-    # This is almost always dev.
-    BRANCH=dev
-    ```
-
-    **If you are copying/pasting commands, make sure you actually set the right
-    value for VERSION above.**
-
-2. Call release preparation helper.
-
-   ```
-   ./etc/bin/release-step-1.sh $VERSION $BRANCH
-   ```
-   
-    This script will:
-    * Create a release branch from the specified branch.
-    * Update CHANGELOG.md with the new version and date.
-    * Update version.go with the new version.
-    * Commit the changes and push the branch to GitHub.
-    * Open a pull request for the release.
-
-3.  Land the pull request after approval as a **merge commit**. To do this,
-    select **Create a merge commit** from the pull-down next to the merge
-    button and click **Merge pull request**. Make sure you delete that branch
-    after it has been merged with **Delete Branch**.
-
-4.  Once the change has been landed, run second script.
-
-   ```
-   ./etc/bin/release-step-2.sh $VERSION $BRANCH
-   ```
-
-   This script will:
-   * Tag the release.
-   * Push the tag to GitHub.
-   * Create a release on GitHub.
-      * (This will open a browser window with the release page. Copy the changelog entries into the release description.)
-   * Switch version and CHANGELOG.md back to development mode.
-   * Commit the changes and push them to GitHub.
-   * Open a pull request for the development changes.
-
-5. Send pull request to a peer review.
-
-# Manual release
+## Releasing
 
 If the above steps fail, you can manually release yarpc-go by following the
 steps below.
 
-(For the sake of simplicity, we will assume that changes are merged to branch `dev`,
-release should be done to branch `master`, and new version is 1.0.0.)
+- Set environment variable with the version you want to release.
 
-1.  Create a release branch from `dev`.
+  ```
+  VERSION=1.0.0
```

Comment:
can we also add how to decide which/how to choose next version before this section?